Maple2.Model/Game/User/Character.cs

```diff
@@ -44,6 +45,7 @@ public class Character {
     public string GuildName = string.Empty;
     public long GuildId;
     public int PartyId;
+    public List<long> ClubIds = []; // ClubId, ClubName
```

Comment:
You can decide, but would `public readonly long[] ClubIds = new long[3];` be more appropriate?

Reply:
Maybe it's just me but I find having this as an array is more of a hassle to add/remove ids in this instance.

Reply:
Fair enough, if the close is cleaner with list it doesn't matter that much